readme.md

```diff
@@ -185,11 +185,14 @@ const unsub1 = useDogStore.subscribe(console.log)
 useDogStore.setState({ paw: false })
 // Unsubscribe listeners
 unsub1()
+// Destroying the store (removing all listeners)
+useStore.destroy()
```

Comment:
Let's remove this as `.destroy()` is deprecated and will be removed in the next major.